INPUT:
  idEmprestimo: 999
  motivo: TESTE
  
OUTPUT:
  _exception: java.lang.IllegalArgumentException
  message: Emprestimo n√£o encontrado com o id 999